### PR TITLE
Outsource actors list to own component

### PR DESCRIPTION
**Pull request by Caruki**
Outsource the actors list of the modal to its own component
Fix some styling on the pictures 
Add scrollbar to show more actors with nav arrow

### CODE REVIEW

File: client/src/components/ActorList.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          <ActorContainer key={actor.name}>
+            <ActorImageWrapper>
+              <img src={actor.image} alt={`picture of ${actor.name}`} />
+            </ActorImageWrapper>
+            {actor.name}
+          </ActorContainer>
+        ))}
+      </AllActorsContainer>
+      {actors[4] && (
+        <GoOn scrolled={scrolled}>
```
[the remark this answers is not in the document]
Comment (Caruki) [reply client/src/components/ActorList.js:104]: it is just an indicatior that you CAN scroll ...... 🤔 no scroll function actually attached to the button. no slider function... you think this is needed here?

[the remark this answers is not in the document]
Comment (Caruki) [reply client/src/components/ActorList.js:104]: 👯‍♀️

[the remark this answers is not in the document]
Comment (marwinburesch) [reply client/src/components/ActorList.js:104]: Aaaah I get it now, well I was clicking it quite a bit :D :D

[the remark this answers is not in the document]
Comment (Caruki) [reply client/src/components/ActorList.js:104]: maybe i will revisit this later. but for now I think it's fine. 😉

**Pull request merged**